src/main/resources/data/minecolonies/quests/general/cookies.json

```diff
@@ -0,0 +1,258 @@
+{
+  "name": "Cookies for the Kids!",
+  "parents": [],
+  "max-occurrences": 1,
+  "triggers": [
+    {
+      "type": "minecolonies:random",
+      "rarity": 250000
+    },
+    {
+      "type": "minecolonies:state",
+      "state": {
+        "path": "buildingManager/buildings",
+        "match": {
+          "type": "minecolonies:baker",
+          "level": 1
+        }
+      }
+    },
+    {
+      "type": "minecolonies:citizen",
+      "state": {
+        "match": {
+          "job": {
+            "type": "minecolonies:baker"
+          }
+        }
+      }
+    },
+    {
+      "type": "minecolonies:citizen",
+      "state": {
+        "match": {
+          "child": true
+        }
+      }
+    }
+  ],
+  "objectives": [
+    {
+      "type": "minecolonies:dialogue",
+      "target": 0,
+      "text": "Hi, Thank you for hiring me as a baker! I always wanted to be a baker! Let's make some cookies for young ones!",
+      "options": [
+        {
+          "answer": "Sure, what do you need?.",
+          "result": {
+            "type": "minecolonies:dialogue",
+            "text": "I need two wheat and one coco bean!",
+            "options": [
+              {
+                "answer": "On my way!",
+                "result": {
+                  "type": "minecolonies:advanceobjective",
+                  "go-to": 1
+                }
+              },
+              {
+                "answer": "They're not my kids, why should I do it?",
+                "result": {
+                  "type": "minecolonies:advanceobjective",
+                  "go-to": -1
+                }
+              }
+            ]
+          }
+        },
+        {
+          "answer": "I'm sorry, I can't help right now.",
+          "result": {
+            "type": "minecolonies:cancel"
+          }
+        },
+        {
+          "answer": "I won't be able to help you with that.",
+          "result": {
+            "type": "minecolonies:advanceobjective",
+            "go-to": -1
+          }
+        }
+      ]
+    },
+    {
+      "type": "minecolonies:delivery",
+      "details": {
+        "target": 0,
+        "item": "minecraft:wheat",
+        "qty": 2,
+        "next-objective": 2
+      }
+    },
+    {
+      "type": "minecolonies:delivery",
+      "details": {
+        "target": 0,
+        "item": "minecraft:cocoa_beans",
+        "qty": 2,
```

Comment:
The baker seems to be asking for one cocoa bean.
Either they are asking for not enough cocoa beans, or this should be `qty: 1`

Reply:
good catch